lang/java/reef-runtime-multi/src/main/java/org/apache/reef/runtime/multi/driver/RuntimesHost.java

```diff
@@ -127,6 +131,17 @@ private void initializeInjector(final Injector runtimeInjector) throws Injection
     runtimeInjector.bindVolatileParameter(
             RuntimeParameters.RuntimeStatusHandler.class,
             runtimeStatusEventHandler);
+    HttpServer httpServer = null;
+    try {
+      httpServer = this.originalInjector.getInstance(HttpServer.class);
+    } catch(InjectionException e) {
+      LOG.log(Level.INFO, "Http Server is not configured for the runtime");
```

Comment:
Log an exception here, e.g.

    LOG.log(Level.WARN, "Http Server is not configured for the runtime", e);